docs/troubleshooting/kubernetes.rst

```diff
@@ -1,13 +1,223 @@
 Troubleshoot Your Kubernetes Deployment
 =======================================
 
-BIG-IP Controller
------------------
+.. toctree::
+   :maxdepth: 1
+
+How to get help
+---------------
+
+If the issue you're experiencing isn't covered here, try one of the following options:
+
+- `Contact F5 Support`_ (valid support contract required).
+- `Report a bug <https://github.com/F5Networks/k8s-bigip-ctlr/issues>`_ in the k8s-bigip-ctlr GitHub repo.
+- `Ask a question <https://f5cloudsolutions.slack.com>`_ in the #cc-kubernetes channel in the F5 Cloud Solutions Slack team.
+
+
+General Kubernetes and OpenShift troubleshooting
+------------------------------------------------
+
+The following troubleshooting docs may help with Kubernetes- or OpenShift-specific issues.
+
+- `Kubernetes: Troubleshoot Applications <https://kubernetes.io/docs/tasks/debug-application-cluster/debug-application/>`_
+- `Kubernetes: Troubleshoot Clusters <https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/>`_
+- `OpenShift: Troubleshooting OpenShift SDN <https://docs.openshift.org/1.5/admin_guide/sdn_troubleshooting.html>`_
+
+.. _k8s-bigip-ctlr troubleshoot:
+
+BIG-IP Controller for Kubernetes/OpenShift
+------------------------------------------
+
+.. _common troubleshoot:
+
+Common Issues
+`````````````
+
+How do I check the settings of my |kctlr| using the command line?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use `kubectl`_ or `oc`_ commands to check the |kctlr| configurations using the command line.
```

Comment:
Do we comment else where that `oc` is only available in Openshift? I think that's worth calling out.

Reply:
I think @jputrino is going to separate k8s & openshift troubleshooting docs

Reply:
Fixed by moving OpenShift into its own doc.